charts/posthog/values.yaml

```diff
@@ -745,6 +746,33 @@ externalStatsd:
   port:
 
 
+###
+###
+### ---- prometheus-kafka-exporter ----
+###
+###
+prometheus-kafka-exporter:
+  # -- Whether to install the `prometheus-kafka-exporter` or not.
+  enabled: false
+
+  # -- We want to pin to image tag `v1.4.2` as it is currently the only
```

Comment:
Is this holding us back a version or pinning past the detail?

Reply:
Nope, I've simply set this to the latest release in order to be able to test this locally on my mac